### PR TITLE
zh-CN: updated the translation of `user-select`

### PR DESCRIPTION
**Pull request by daiwanxing**

### CODE REVIEW

File: files/zh-cn/web/css/user-select/index.md [hunk 1]
```diff
@@ -5,44 +5,31 @@ slug: Web/CSS/user-select
 
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/Web/CSS) 属性 **`user-select`** 控制用户能否选中文本。除了文本框内，它对被载入为 {{Glossary("Chrome", "chrome")}} 的内容没有影响。
+`user-select` CSS 属性用于控制用户是否可以选择文本。这不会对作为浏览器用户界面（即 {{Glossary("Chrome", "chrome")}}）的一部分加载的内容产生任何影响，除非是在文本框中。
```
Comment (yin1999) [inline files/zh-cn/web/css/user-select/index.md:8]: Hi @daiwanxing, you have removed the bold style and link to CSS landing page

Comment (daiwanxing) [reply files/zh-cn/web/css/user-select/index.md]: 已修复此错误

File: files/zh-cn/web/css/user-select/index.md [hunk 2]
```diff
@@ -5,44 +5,31 @@ slug: Web/CSS/user-select
 
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/Web/CSS) 属性 **`user-select`** 控制用户能否选中文本。除了文本框内，它对被载入为 {{Glossary("Chrome", "chrome")}} 的内容没有影响。
+**`user-select`** CSS 属性用于控制用户是否可以选择文本。这不会对作为浏览器用户界面（即 {{Glossary("Chrome", "chrome")}}）的一部分加载的内容产生任何影响，除非是在文本框中。
```
Comment (yin1999) [inline files/zh-cn/web/css/user-select/index.md:8]:
```suggestion
**`user-select`** [CSS](/zh-CN/docs/Web/CSS) 属性用于控制用户是否可以选择文本。这不会对作为浏览器用户界面（即 {{Glossary("Chrome", "chrome")}}）的一部分的内容加载产生任何影响，除非是在文本框中。
```

File: files/zh-cn/web/css/user-select/index.md [hunk 3]
```diff
@@ -5,44 +5,31 @@ slug: Web/CSS/user-select
...
 
-{{cssinfo}}
-
 > **备注：** `user-select` 不是继承属性，即使默认的属性值 `auto` 的表现基本上以继承为主，似乎是继承属性。甚至，WebKit/基于 Chromium 的浏览器在实现此属性时*将其作为继承属性*，但这和有关规范是相悖的，且会带来一些问题。目前，Chromium 暂时选择[修复将其作为继承属性所带来的问题](https://chromium.googlesource.com/chromium/src/+/b01af0b296ecb855aac95c4ed335d188e6eac2de)，使最终表现符合规范。
 
-## 语法
+### 取值
 
 - `none`
   - : 元素及其子元素的文本不可选中。请注意这个{{domxref("Selection")}} 对象可以包含这些元素。从 Firefox 21 开始， `none` 表现的像 `-moz-none`，因此可以使用 `-moz-user-select: text` 在子元素上重新启用选择。
```
Comment (yin1999) [inline files/zh-cn/web/css/user-select/index.md:35]:
```suggestion
  - : 元素及其子元素的文本不可选中。请注意，{{domxref("Selection")}} 对象可以包含这些元素。
```

File: files/zh-cn/web/css/user-select/index.md [hunk 4]
```diff
@@ -66,9 +53,13 @@ user-select: unset;
 - `element`{{non-standard_inline}}（IE 专有别名）
   - : 与 `contain` 相同，但仅在 Internet Explorer 中受支持。
```
Comment (yin1999) [inline files/zh-cn/web/css/user-select/index.md:54]:
```suggestion
```

File: files/zh-cn/web/css/user-select/index.md [hunk 5]
```diff
@@ -66,9 +53,13 @@ user-select: unset;
...
 
-> **备注：** CSS UI 4 已[将 `user-select` 的 `element` 属性值重命名为 `contain`](https://github.com/w3c/csswg-drafts/commit/3f1d9db96fad8d9fc787d3ed66e2d5ad8cfadd05)。
+    > **备注：** CSS UI 4 将 `element` 值[重命名](https://github.com/w3c/csswg-drafts/commit/3f1d9db96fad8d9fc787d3ed66e2d5ad8cfadd05) `contain`。
```
Comment (yin1999) [inline files/zh-cn/web/css/user-select/index.md:56]:
```suggestion
    > **备注：** CSS UI 4 将 `element` 值[重命名](https://github.com/w3c/csswg-drafts/commit/3f1d9db96fad8d9fc787d3ed66e2d5ad8cfadd05)为 `contain`。
```

File: files/zh-cn/web/css/user-select/index.md [hunk 6]
```diff
@@ -114,6 +101,6 @@ user-select: unset;
 
 ## 参见
 
-- {{Cssxref("::selection")}}
+- [为 `user-select: contain` 做 polyfill](https://github.com/github/user-select-contain-polyfill)
```
Comment (yin1999) [inline files/zh-cn/web/css/user-select/index.md:104]:
```suggestion
- [`user-select: contain` 的 polyfill](https://github.com/github/user-select-contain-polyfill)
```

**Review by yin1999 — COMMENTED** [review]
Why not removing those out-of-date content

**Review by yin1999 — APPROVED** [review]

**Pull request merged**